src/components/Tables/WeatherTable.jsx

```diff
@@ -2,29 +2,30 @@ import { useContext } from 'react';
 import { DataContext } from '../../contexts/DataContext';
 import WeatherInput from '../Inputs/WeatherInput';
 import './tables.scss';
+import WeatherTableHeader from './WeatherTableHeader';
 
 const RainfallTable = () => {
 
   const {weatherValues} = useContext(DataContext);
 
   return (
-      <table className="table">
-        <tbody>
-          <tr>
-            <th>Day of the Week</th>
-            <th>Rainfall Amount</th>
-            <th>Wind Speed</th>
-          </tr>
-          {weatherValues.map(({ date, Precipitation, WindSpeed }) => (
-              <tr key = {`${date} + ${Precipitation}`}>
-                <td>{date}</td>
-                <td>{Precipitation}</td>
-                <td>{WindSpeed}</td>
-              </tr>
-              ))}
-          <WeatherInput />  
-        </tbody>
-      </table>
+    <>
+      <WeatherTableHeader />
```

Comment:
This is an additional table, not connected to the one below.  Meaning if we add to the table below, this table is now invalid.  Instead, can we keep an array with the order of values ie:
```
//Somewhere global like constants.js, other things in your app may access this:
const PRECIPITATION_KEY = 'Precipitation';
const WIND_KEY = 'WindSpeed';
const DATE_KEY = 'date';

const tableOrder = [PRECIPITATION_KEY, WIND_KEY, DATE_KEY];

const tableDescriptions = Object.freeze({
  [PRECIPITATION_KEY]: 'Rainfall amount'
  ....
});
```
That way we could dynamically add the td's below without having to deconstruct the whole object or add to a bunch of files if we end up with another field.  And we could pass our header the same info and it could dynamically create the table from that (and throw an error if there's no description available)

Also the reason why I'm making constants for the keys right now is because we seem to be basing the names off of what is printed in the graph.  It would be better if Precipitation and WindSpeed were precipitation and windSpeed since this is JavaScript which is usually camelCased.  And it would be better if we updated the c3 chart to spit out Precipitation instead of the variable name.